lib/public/views/Runs/Overview/RunsOverviewModel.js

```diff
@@ -51,6 +54,15 @@ export class RunsOverviewModel extends OverviewPageModel {
         this._eorReasonsFilterModel.observe(() => this._applyFilters());
         this._eorReasonsFilterModel.visualChange$.observe(() => this.notify());
 
+        this._runDurationFilterModel = new DurationInputModel();
```

Comment:
runDurationFilterModel must somehow store the operator and the duration.
Maybe you should create a class `DurationFilterModel` that group both.

Reply:
applied